src/fixes/aiming_after_equip.cpp

```diff
@@ -14,18 +14,28 @@ void FixAimAfterEquip::Initialize()
 
     if (eGameType & MGS2)
     {
-        MAKE_HOOK_MID(baseModule, "89 1D ?? ?? ?? ?? 8B C3 48 83 C4", "MGS2: Aiming After Equip", {
+        MAKE_HOOK_MID(baseModule, "89 1D ?? ?? ?? ?? 8B C3 48 83 C4", "MGS2: Aiming After Equip - Snake", {
             g_GameVars.SetAimingState(0);
-        })
-    }
+        });
+
+        MAKE_HOOK_MID(baseModule, "89 B3 ?? ?? ?? ?? 48 8B 83 ?? ?? ?? ?? 4C 8B 74 24, "MGS2: Aiming After Equip - Raiden", {
+            if ((uint32_t)ctx.rsi == 0)
+            {
+             g_GameVars.SetAimingState(0);
+           }
+        });
+
+        }
+    
```

Comment:
```suggestion
```suggestion
        MAKE_HOOK_MID(baseModule, "89 B3 ?? ?? ?? ?? 48 8B 83 ?? ?? ?? ?? 4C 8B 74 24, "MGS2: Aiming After Equip - Raiden", {
            if ((uint32_t)ctx.rsi == 0)
            {
                g_GameVars.SetAimingState(0);
            }
        });
        }
    
```